ROADMAP.md

```diff
@@ -78,16 +78,25 @@ Please file an issue to make suggestions on this roadmap!
 
 ## v0.9.10
 
-  * kubeadm support to simplify k8s components configuration (#654)
+  * Kubernetes 1.9.x
+  * Security+Usability improvements
+    * [kiam](https://github.com/uswitch/kiam/) integration (#1055)
+    * [authenticator](https://github.com/heptio/authenticator) integration (#1153)
+    * Support for pregenerating IAM roles used by kube2iam/kiam (#1145, #1150)
+  * Operatability improvements
+    * [More manageable Calico + Flannel](https://github.com/kubernetes-incubator/kube-aws/pull/675#issuecomment-303669142) (@redbaron) (#909)
+    * Graduate from relying on CloudFormation nested stacks (#1112)
+    * Ease certificate rotation (#1146)
+
+## v0.9.11
+
+  * Kubernetes 1.10
+  * (After easy H/A controller support) kubeadm support to simplify k8s components configuration (#654)
     * Reduces the amount of code required in kube-aws
     * To better follow upstream improvements on how k8s components are deployed
-  * [More manageable Calico + Flannel](https://github.com/kubernetes-incubator/kube-aws/pull/675#issuecomment-303669142) (@redbaron)
-  * istio integration
+  * (After scalability/reliability/upgradability cleared) istio integration
     * Probably after k8s supported injecting init containers from PodPreset
       * [Upstream issue](https://github.com/kubernetes/kubernetes/issues/43874)
-
-## v0.9.11
-
   * Migrate from coreos-cloudinit to ignition for node bootstrapping (@redbaron)
```

Comment:
Honestly, I don't have specific use-cases require ignition.

Can't we just use bash rather than coreos-cloud-init or ignition for most of node configuration steps? :)
The goal of node configuration is just reliably fetching assets from S3 and configuring systemd units.

Note that even with that direction, we could make ignition rather than cloud-init to be the entrypoint and just call the kube-aws's bash script for node configuration. So that users can still modify ignition config directly when necessary.

I'm just saying that we don't necessarily migrate all the things we've been doing in cloud-config to ignition.